server/db/dbcontrollers/trades.js

```diff
@@ -0,0 +1,176 @@
+var methods = {};
+
+module.exports = function(knex) {
+
+  //think about using a Knex transactions to lock down the price while query happening post MVP thing
+  //think about checking that they have less than or equal to 25 stocks post MVP
+
+  //-------------------------------------buy a stock controller -----------------------------------------//
+  methods.buy = function (userID, matchID, numShares, action, stockTicker) {
+    var stockID;
+    var total;
+    var cashRemaining;
+    var cashToDate;
+    var price;
+      
+    //get the stock id
+    return knex.select().from('stocks').where('symbol','=', stockTicker)
+    .then(function(stock) {
```

Comment:
we don't really use the stockId for anything anymore, so this can be refactored out.

Front end is expecting symbol instead of stockId